pkg/generator.go

```diff
@@ -461,6 +473,10 @@ type namer interface {
 }
 
 func (g *Generator) renderType(ctx context.Context, typ types.Type) string {
+	if replacement, ok := g.config.ReplaceGeneric[typ.String()]; ok {
+		return replacement
+	}
+
```

Comment:
To be semantically correct, what this is actually doing is to replace any type (not just generics). I wonder if we should instead change the name of the parameter to be something like `replace-param-type`?

I need to think about this carefully because we already have `replace-types` which _should_ be able to handle this, but that feature I don't think works with generics. I'm open to ideas.

Reply:
You're right, it should use replace-types. I'll update the PR this week!